### PR TITLE
[FUSETOOLS2-1653] Upgrade strip-ansi to 7.0.1

### PR DESCRIPTION
**Pull request by joshiraez**
Fixes https://issues.redhat.com/browse/FUSETOOLS2-1653

Resources: 
* https://github.com/aws/aws-toolkit-vscode/issues/2285
* https://github.com/aws/aws-toolkit-vscode/pull/1775
* https://www.typescriptlang.org/tsconfig#esModuleInterop

Strip-ansi 7.0 is a pure ES module so there is a need to enable esModuleInterop. This causes that we need to change some imports on those modules that are not ES6 modules (those whose files don't have the export ___ in the end) to not use the `import * as ____ from ____` and instead put those imports as `import ____ from ____`

This PR adds the esModuleInterop compiler version, fixes the imports that break after it, and bumps the version in strip-ansi

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,8249 @@
 {
 	"name": "vscode-camelk",
 	"version": "0.0.32",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
```
Comment (apupier) [inline package-lock.json:4]: the package-lock.json wasn't upgraded yet to a recent npm versiobn. it would have been better to do the upgrade in another PR as it will allow to review the changes. Fine for this time.

File: package.json [hunk 2]
```diff
@@ -399,10 +399,16 @@
...
 		"tar": "^6.1.11",
 		"tmp": "^0.2.1",
 		"valid-filename": "^3.1.0",
 		"vscode-kubernetes-tools-api": "^1.3.0"
+	},
+	"__metadata": {
+		"id": "5181e15d-d18d-444e-ba3b-e0cf48cab6f8",
+		"publisherDisplayName": "Red Hat",
+		"publisherId": "eed56242-9699-4317-8bc7-e9f4b9bdd3ff",
+		"isPreReleaseVersion": false
```
Comment (apupier) [inline package.json:412]: why is it added?

File: test Fixture with speci@l chars/TestDeployInSpecificNamespace.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+      // Write your routes here, for example:
+      from("timer:java?period=1000")
+        .routeId("java")
+        .setBody()
+          .simple("Hello Camel K from ${routeId}")
+        .to("log:info");
+
+  }
+}
```
Comment (apupier) [inline test Fixture with speci@l chars/TestDeployInSpecificNamespace.java:17]: I think this file is unrealted to this change

**Review by apupier — CHANGES_REQUESTED** [review]
good findings with the esModuleInterOp

File: test Fixture with speci@l chars/TestDeployInSpecificNamespace.java (hunk 3 above)
Comment (joshiraez) [reply test Fixture with speci@l chars/TestDeployInSpecificNamespace.java:17]: Wops my bad. I'll remove it

File: package.json (hunk 2 above)
Comment (joshiraez) [reply package.json:412]: I'm not sure. I'll remove it and try if it breaks anything. It might be some leftover of some node script.

**Review by apupier — APPROVED** [review]
